README.md

```diff
@@ -112,6 +112,24 @@ You can check [http://127.0.0.1:8080/api/status/v1/status](http://127.0.0.1:8080
 
 You can also check [http://0.0.0.0:8432](http://0.0.0.0:8432) which should show 'Ok' if the datastore emulator is working properly.
 
+### Run locally from IntelliJ
+
+These instructions are for running the app in IntelliJ specifically, not PyCharm.
+
+- Go to `Run` -> `Edit Configurations` -> `Add New Configuration` -> `Flask Server`
+- For `Target Type` choose: `Module Name`
+- In the `Target` field enter: `main.py`
+- Leave the `Application`, `Additional Options`, and `FLASK_ENV` fields blank
+- If you want to debug the application, check the `FLASK_DEBUG` checkbox
+- In the `Environment variables` field enter: `DATASTORE_EMULATOR_HOST=0.0.0.0:8432;FLASK_HOST=127.0.0.1`
+- Click `OK`
+- Try running the app by clicking `Run` -> `Run 'Flask (main.py)'` (or by clicking the `Run` button in the Toolbar)
```

Comment:
I also needed to set up my python interpreter. (first I used pyenv to set up my virtualenv. Then I used this guide to set up my python interpreter in intellij: https://www.jetbrains.com/help/idea/creating-virtual-environment.html)


The server successfully started up  after I installed project dependencies and rendered configs following the steps in the readme above.